package.json

```diff
@@ -49,7 +49,7 @@
     "test:watch": "jest --notify --watch",
     "test:ci": "jest --runInBand",
     "tdd": "yarn run test:watch",
-    "format:js": "prettier --write \"{lib,tests}/**/*.js\"",
+    "format:js": "prettier --write \"{lib,tests,stories/webapp}/**/*.js\"",
```

Comment:
Should we format all stories? Maybe we should, I'll change this but in another PR so I don't make this MASSSSIIIVEEE!

Reply:
👍

Reply:
Yeah, prolly lint as well. Although I wouldn't want to be the person to fix all the lint errors!